maturity-level-one/tests/Codit.IntegrationTest/PlayersTest.cs

```diff
@@ -0,0 +1,227 @@
+using System;
+using Xunit;
+using Microsoft.AspNetCore.TestHost;
+using Microsoft.AspNetCore.Hosting;
+using System.Net.Http;
+using System.Net;
+using System.Threading.Tasks;
+using Codit.LevelOne.Models;
+using Newtonsoft.Json;
+using System.Text;
+using System.Net.Http.Headers;
+using Microsoft.AspNetCore.JsonPatch;
+
+namespace Codit.IntegrationTest
+{
+    public class PlayersTest
+    {
+        private readonly HttpClient _httpClient;
+        public PlayersTest()
+        {
+            if (_httpClient != null) return;
+            var srv = new TestServer(new WebHostBuilder()
+                .UseEnvironment("Development")
+                .UseStartup<Codit.LevelOne.Startup>());
+
+            _httpClient = srv.CreateClient();
+        }
+
+        [Theory]
+        [InlineData("GET")]
```

Comment:
For all of these - Given you only test it with GET, why not move it inside the test in arrange part?

Reply:
will be done in a separate PR

Reply:
See other comment - Let's just bite the bullet and do it now